menu.py

```diff
@@ -26,6 +27,7 @@ def menu():
 [3] Usuń produkt z koszuka
 [4] Mini Quiz o tematyce BIO
 [5] Dodaj produkt do magazynu
+[6] Sortuj produkty
```

Comment:
When I get to insert the wrong type of option (e.g. pressing a number instead of a supported letter),  it gets me immediately to MAIN menu instead of  to the "Sortuj produkty"  option and I get to start over again.